Importer/CsvImporter.php

```diff
@@ -193,9 +173,9 @@ protected function importData(array $parsedData, $dataIsAssociativeArray, $flush
 
         try {
             $this->entityManager->flush();
-        } catch (ORMException $exception) {
-            $message = 'Error when flushing for entity.';
-            $this->handleImporterError(new Error($message, $exception));
+        } catch (\Exception $exception) {
+            $error = new Error('Error during flush', $exception);
```

Comment:
Have you found any articles about using events to handle errors in processes?

I mean, it doesn't look bad, an error *is* an event so why to handle it with event listener, but I'm just asking because maybe it's one of those things (probably not, but...) that you have to learn not to do in a hard way, or to listen someone else's advice "don't do it because this and that".